dotnet/dotnet-hello-world/src/Pages/_ViewImports.cshtml

```diff
@@ -0,0 +1,3 @@
+@using src
+@namespace src.Pages
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
```

Comment:
Using is typically means "adding a package". So it should be a package name, not the folder name.
Package name in this case is likely dotnet_hello_world or the like